src/components/QuoteGenerator.vue

```diff
@@ -195,7 +197,49 @@ const displayDoorHeights = computed((): Array<string> => {
 function updateExtrasList(extra: string): void {
   selectedExtras.value.push(extra);
 }
+
+const panelPartNumber = computed((): string => {
+
+      // Define the bulk panel prefix
```

Comment:
Fix the indentation in lines 203 -> 224 (use two blankspaces and not tabs to keep the consistensy across all editors out there).
And feel free to merge the PR.